soft_999_Jenish_Patel_fw_102

```diff
@@ -0,0 +1,45 @@
+// Blink an LED
+#include <stdint.h>
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+#define HALF_SECOND 500  //
```

Comment:
since this is a value in milliseconds, our convention is to suffix it with `_MS`, please rename